core/src/Kokkos_ExecPolicy.hpp

```diff
@@ -40,6 +40,35 @@ struct ChunkSize {
 #endif
 };
 
+namespace Impl {
+
+template <class ExecType>
+concept ExecSpaceConcept = Kokkos::is_execution_space_v<ExecType>;
```

Comment:
See https://github.com/kokkos/kokkos/pull/8494.

Reply:
I updated the names to match that PR, so should be seamless rebase.